cli/lib/kontena/cli/log_formatters/compact.rb

```diff
@@ -0,0 +1,55 @@
+require 'logger'
+
+module Kontena
+  module Cli
+    module LogFormatter
+      class Compact < Logger::Formatter
+        def self.ms_since_first
+          ((Time.now.to_f - @first_log) * 1000).to_i
+        end
+
+        def self.ms_since_last
+          ((Time.now.to_f - @last_log) * 1000).to_i
+        ensure
+          @last_log = Time.now.to_f
+        end
+
+        def self.__init_timers__
+          @first_log = Time.now.to_f
+          @last_log = @first_log
+        end
+
+        __init_timers__
+
+        def colorize_severity(severity)
+          case severity[0..0]
+          when 'D' then Kontena.pastel.blue('D')
+          when 'W' then Kontena.pastel.yellow('W')
+          when 'I' then Kontena.pastel.cyan('I')
+          when 'E', 'F' then Kontena.pastel.red('E')
+          else severity[0..0]
+          end
+        end
+
+        def colorize_time
+          elapsed = self.class.ms_since_last
+          str = sprintf("%6s", "#{elapsed}ms")
```

Comment:
> Can we nail down the format, the code itself looks good.
> 
> The current format (ms since last log entry) is kinda handy to see the rough execution times, but as stated in above comments it can also be bit misleading.

My suggestion would be something like `"%7.3f" % (Time.now.to_f - @first_log)`. Could still keep the coloring based on the `ms_since_last ` thresholds?